### PR TITLE
switch from babel-preset-es2015 to babel-preset-env

### PR DESCRIPTION
**Pull request by svenkatreddy**
- switch from babel-preset-es2015 to babel-preset-env
- update dynamic examples to use .babelrc for babelify (easy to maintain)
- add contributors section as mentioned in previous PR
- Addresses #112 and #113

### CODE REVIEW

**Comment by svenkatreddy** [comment]
@zpao

File: index.js [hunk 1]
```diff
@@ -20,7 +20,11 @@ var DEFAULT_OPTIONS = {
   babel: {
     presets: [
       'react',
-      'es2015',
+      [ "env", {
+        "targets": {
+          "node": "current",
+        },
+      }],
```
Comment (zpao) [inline index.js]: Can you format this to match the rest of the file: newline inside the array, use single quotes.

File: README.md [hunk 2]
```diff
@@ -149,18 +149,24 @@ All "locals" are exposed to your view in `this.props`. These should work identic
...
   // is a common use
 `}} />
 ```
 
 * It's not possible to specify a doctype in JSX. You can override the default HTML5 doctype in the options.
 
+## Contributors
+* [Zpao][zpao] (Author)
+* [Venkat Reddy][svenkatreddy]
+
```
Comment (zpao) [inline README.md]: Please keep your PRs to a single purpose and do this separately, it's entirely unrelated to switching to babel-env.

Also, please use my name (`Paul O’Shannessy`)

**Comment by zpao** [comment]
Ok, made some changes so that you don't have to think about formatting much. Please rebase on top of master to pick that up.

Also, missed this before but please use `yarn` so that `yarn.lock` is updated

File: README.md (hunk 2 above)
Comment (svenkatreddy) [reply README.md]: noted! removed, will open a separate PR for that

**Comment by svenkatreddy** [comment]
Thanks for adding eslint to master.
i rebased and re-ran prettier.

**Comment by zpao** [comment]
Thanks!

**Comment by svenkatreddy** [comment]
Thanks for reviewing again.
Are you planning on releasing new version with this change ?

**Pull request merged**